src/main/java/com/spring/config/AppConfig.java

```diff
@@ -4,24 +4,35 @@
 import java.util.Properties;
 import javax.sql.DataSource;
 import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
 import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
 
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "com.spring.dao",
         "com.spring.service"
 })
 public class AppConfig {
+    private final Environment environment;
+
+    @Autowired
```

Comment:
in previous HW you were not using @Autowired annotation during constructor injection, let's keep it this way(or add her everywhere)